BaSP-a2022/ayom-a-trackgenix-app
# BaSP-TG-95: Fix bug display modal in entities

Add close modal function after create or edit entities.

QA:
When you navigate to an entity after create or edit, should not display any modal!
